### DISCUSSION TITLE
[✨] Modify head at runtime with useDocumentHead

### DISCUSSION THREAD

**Discussion by GrandSchtroumpf** [QwikDev/qwik-evolution#142] [Proposals For Qwik]
### Is your feature request related to a problem?

I want to update the head content dynamically from a hook. I've got a theme picker and want to update the [theme-color](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name/theme-color) meta tag.
From what I understand this is currently impossible to update head properties at runtime.

### Describe the solution you'd like

Two options : 
1. `useDocumentHead().meta` can be a `Signal`.
2. `useDocumentHead()` could provide an `update` method.

I would rather use option 1.

### Describe alternatives you've considered

Use the DOM api, but then it wouldn't work on the server.

**Comment by ahnpnl** [comment]
I have another alternative is, you can create a global context and update context in component. In `RouterHead`, you can react to the context change to set meta tag value and combine with existing meta tags from `useDocumentHead`, inspired by this https://robisim74.gitbook.io/qwik-speak/library/quick-start#head-metas

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
